### ISSUE TITLE
No filtro en firebase.net 

### ISSUE THREAD

**Issue by Elsaviactivo** [step-up-labs/firebase-database-dotnet#288]
the library does not allow to search in a list of elements by child as in android. for example : order by child

**Comment by bezysoftware** [comment]
See https://github.com/step-up-labs/firebase-database-dotnet#querying

**Comment by RonalPortes** [comment]
Yes you can. Check the query instructions. Also make sure you index the child you want to query on firebase.
